### DISCUSSION TITLE
Valvola termostatica aqara E1, associare  sensore temperatura aqara

### DISCUSSION THREAD

**Discussion by alenyc81** [indomus/forum#2758] [HUB: Home Assistant]
Ciao, qualcuno sa come accoppiare in home assistant il sensore temperatura aqara alla valvola termostatica aqara E1, di modo da non usare il termometro interno della valvola?

**Comment by snakuzzo** [comment]
Dai una letta a questa discussione -> https://github.com/indomus/forum/discussions/2739 sono concetti che valgono anche per il tuo caso.
In sintesi...puoi associare le due cose con un generic thermostat ma, per far funzionare tutto correttamente, se la testa nativamente non consente l'associazione di un sensore esterno, deve almeno consentire l'impostazione di un offset che dovrai far sì che venga modificato dinamicamente in base alla temperatura rilevata dal sensore esterno. Questo perché la testa ha una sua logica interna e, per quanto tu possa forzare alcune cose lato HA, "vince" sempre quella.

**Reply by alenyc81** [reply]
Ok grazie, cmq nativa può essere associata al sensore esterno aqara... Al momento le due termostatiche montate le sto usando con un automazione di on off in base al raggiungimento di una certa temperatura del sensore... Di fatto non è che cambia molto...

**Reply by snakuzzo** [reply]
ok...quindi non capisco perché ti serve HA per fare questa associazione che di fatto è già in essere

**Reply by alenyc81** [reply]
Perché le valvole devono essere associate al sensore esterno... E si associano con l app aqara,ma serve l hub, oppure in HA

**Reply by snakuzzo** [reply]
https://www.zigbee2mqtt.io/guide/usage/binding.html
